### PR TITLE
REturn to netzkarte Popup on departure popup close with close button.

### PR DESCRIPTION
**Pull request by edellucien**
Allow to redirect to netzkarte popup, when closing departure popup.

### CODE REVIEW

File: src/components/FeatureInformation/FeatureInformation.js [hunk 1]
```diff
@@ -82,6 +82,9 @@ const FeatureInformation = ({ clickedFeatureInfo }) => {
             className="wkp-close-bt"
             onClick={() => {
               dispatch(setClickedFeatureInfo());
+              if (PopupComponent.actionOnClose) {
+                PopupComponent.actionOnClose();
```
Comment (oterral) [inline src/components/FeatureInformation/FeatureInformation.js]: DidUnmount  is not called on DeparturePopup when we close the popup with the close button ?

Comment (edellucien) [reply src/components/FeatureInformation/FeatureInformation.js]: 'componentWillUnmount' is called when clicking on the close button, but as well on clicking outside of the popup.
This solution allows us to return to the netzkarte Popup only on click on close button.

File: src/popups/DeparturePopup/DeparturePopup.js [hunk 2]
```diff
@@ -24,5 +48,9 @@ DeparturePopup.defaultProps = defaultProps;
 
 const composed = compose(withTranslation())(DeparturePopup);
 composed.renderTitle = feat => feat.get('name');
+// Trigerred on popup close with close button only.
+composed.actionOnClose = () => {
```
Comment (oterral) [inline src/popups/DeparturePopup/DeparturePopup.js]: The namimg is a bit confusing since it applies only when we click on the close button.

**Comment by edellucien** [comment]
Because the onCloseClick is called when we click on the close button in the popup's Header.
However here, we are hidding the header in the popup (because we don't want the default one), and adding the header in the FeatureInformation component.

File: src/popups/DeparturePopup/DeparturePopup.js (hunk 2 above)
Comment (edellucien) [reply src/popups/DeparturePopup/DeparturePopup.js]: You are right, I renamed it 'onCloseBtClick'.

**Pull request merged**